multiset.go

```diff
@@ -0,0 +1,97 @@
+package secp256k1
+
+// #include "./depend/secp256k1/include/secp256k1_multiset.h"
+import "C"
+import "github.com/pkg/errors"
+
+// MultiSet is a type used to create an Elliptic Cure Multiset Hash.
+// which is a rolling(homomorphic) hash that you can add and remove elements from
+// and receiving the same resulting hash as-if you never hashed that element.
+// Because of that the order of adding and removing elements doesn't matter.
+// The type should only be created via the supplied methods.
+type MultiSet struct {
+	set C.secp256k1_multiset
+}
+
+// NewMultiset return an empty initialized set.
+// when finalized it should be equal to a finalized set with all elements removed.
+func NewMultiset() MultiSet {
```

Comment:
NewMultiset should return pointer to MultiSet (Unless there's some good reason not to that I'm missing)

Reply:
Why a pointer though? (don't quite get when does Go expects you to return a pointer and when by value)